### DISCUSSION TITLE
showImagePreview函数无法切换图片

### DISCUSSION THREAD

**Discussion by zazahui-hugh** [youzan/vant#12801] [Q&A]
vant4+vue3 使用showImagePreview函数调用时多张图片无法切换图片，预览正常

**Comment by zazahui-hugh** [comment]
1. vant版本4.8.11；
2. 组件方式切换没问题；
3. 函数调用时预览正常，但无法切换

**Reply by inottn** [reply]
可以提供一下复现链接

**Reply by zazahui-hugh** [reply]
已解决，好像是重复引入showImagePreview函数造成的
